specifyweb/frontend/js_src/lib/components/Header/NotificationRenderers.tsx

```diff
@@ -119,6 +126,74 @@ export const notificationRenderers: IR<
       />
     );
   },
+  'record-merge-starting'(notification) {
+    const tableName = notification.payload.table;
+    const collectionId = parseInt(notification.payload.collection_id);
+    const mergeName = notification.payload.name;
+    const collection = userInformation.availableCollections.find(
+      ({ id }) => id === collectionId
+    );
+    const [collectionName, setCollectionName] = React.useState<
+      string | null | undefined
+    >('');
+
+    React.useEffect(() => {
+      setCollectionName(collection?.collectionName);
+    }, [collection]);
+
+    return (
+      <>
+        {mergingText.mergingHasStarted()}
+        <div className="flex items-center gap-2">
+          <TableIcon label name={tableName} />
+          <p>
+            {collectionName} {'-'} {mergeName}
```

Comment:
Relying on JSX for spaces is very unsafe as Prettier does not try to preserve spaces when reformatting JSX, thus this can break in the future.

```suggestion
            {`${collectionName} - ${mergeName}`}
```